### ISSUE TITLE
Doesn't seem to work with m2 max and ventura 13.2.1

### ISSUE THREAD

**Issue by ospfranco** [WCHSoftGroup/ch34xser_macos#19]
I just installed the driver, I can see on the system report that the vendor id is correct. But when I execute `ls tty*` on `/dev/` then nothing pop-ups.

<img width="1227" alt="CleanShot 2023-02-14 at 14 38 32" src="https://user-images.githubusercontent.com/1634213/218754677-659c143d-326b-4f50-a01a-cc1befc18798.png">

<img width="1298" alt="CleanShot 2023-02-14 at 14 38 50" src="https://user-images.githubusercontent.com/1634213/218754770-a5c2e37f-b34a-4282-ab6f-37bf00f3f331.png">

The machine is a new MacBook 14' m2 max with the latest Ventura version (13.2.1).

**Comment by fdesoye** [comment]
Similar issue here: I'm on an M1 with Ventura 13.2.1 and don't even get to see the usb serial device.

**Comment by ospfranco** [comment]
I have an m2 air and there it is working fine, it is on a new machine that it doesn't work.

**Comment by ospfranco** [comment]
Doing a `systemextensionsctl list` command gives:

```
/Users/osp
systemextensionsctl list
4 extension(s)
--- com.apple.system_extension.network_extension
enabled	active	teamID	bundleID (version)	name	[state]
*	*	TC3Q7MAJXF	com.adguard.mac.adguard.network-extension (2.9.1/1217)	AdGuard Network Extension	[activated enabled]
*	*	J6S6Q257EK	ch.protonvpn.mac.OpenVPN-Extension (3.0.11/2212131301)	ProtonVPN OpenVPN	[activated enabled]
*	*	J6S6Q257EK	ch.protonvpn.mac.WireGuard-Extension (3.0.11/2212131301)	ProtonVPN WireGuard	[activated enabled]
--- com.apple.system_extension.driver_extension
enabled	active	teamID	bundleID (version)	name	[state]
*	*	5JZGQTGU4W	cn.wch.CH34xVCPDriver (1.0/1)	cn.wch.CH34xVCPDriver	[activated enabled]
```

I can see the driver installed and active however the device still does not appear when I do `ls /dev/tty*`.

**Comment by ospfranco** [comment]
Managed to fix the issue, apparently when I installed the driver I was too fast and did not enable the extension on the system settings app. Deleting the app from the Applications folders (and all the `rm` commands) was not enough. I had to disable SIP, then manually uninstall the driver via command:

`sudo systemextensionsctl uninstall 5JZGQTGU4W cn.wch.CH34xVCPDriver`

Rebooted the system, installed the driver again (this time I got the system settings prompt and promptly clicked on it and allowed it), rebooted and now it is finally working.
